lib/Dialect/Calyx/Export/CalyxEmitter.cpp

```diff
@@ -1019,11 +1024,15 @@ void Emitter::emitLibraryFloatingPoint(Operation *op) {
     return;
   }
 
-  StringRef opName = op->getName().getStringRef();
+  std::string opName;
+  if (isa<calyx::AddFNOp>(op))
```

Comment:
Same here.

Reply:
Still not a fan of this `if-else`, it implies that there will only ever exist two FN functions, add and mul.